apps/dashboard/src/components/conditions-editor/field-selector.tsx

```diff
@@ -0,0 +1,25 @@
+import { useMemo } from 'react';
+import { FieldSelectorProps } from 'react-querybuilder';
+
+import { Code2 } from '@/components/icons/code-2';
+import { VariableSelect } from '@/components/conditions-editor/variable-select';
+
+export const FieldSelector = ({ handleOnChange, options, ...restFieldSelectorProps }: FieldSelectorProps) => {
```

Comment:
This component represents the first `field` input. It's a input + dropdown with a variables suggestions. The implementation is in the `VariableSelect` component that can be potentially reused in the other places in the future.

![Screenshot 2025-01-14 at 12 30 06](https://github.com/user-attachments/assets/8b2be65e-b418-4d12-86b6-ec0f436b22cf)